applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -180,8 +182,12 @@ get_fs_app(Node, UUID, JObj, <<"record_call">>) ->
             case wh_json:get_value(<<"Record-Action">>, JObj) of
                 <<"start">> ->
                     FollowTransfer = wh_json:get_binary_boolean(<<"Follow-Transfer">>, JObj, <<"true">>),
+                    ConfigSampleRate = ecallmgr_config:get_integer(<<"record_sample_rate">>, ?DEFAULT_SAMPLE_RATE),
+                    SampleRate = wh_json:get_integer_value(<<"Record-Sample-Rate">>, JObj, ConfigSampleRate),
```

Comment:
@jamesaimonetti 
If we talk about data types I'd go this way. It helps to be sure that we use appropriate values on every step